demos/spark/README.md

```diff
@@ -0,0 +1,54 @@
+## Spark 2.4.3 local test:
+```
+1. enter the 0.23 occlum container
+2.apt-get update &&/
+  apt-get install -y openjdk-11-jdk
+3. wget https://archive.apache.org/dist/spark/spark-2.4.3/spark-2.4.3-bin-hadoop2.7.tgz
+4. tar -xvzf spark-2.4.3-bin-hadoop2.7.tgz
+5. replace the spark-network-common_2.11-2.4.3.jar
```

Comment:
Where is `spark-network-common_2.11-2.4.3.jar`? You only upload `spark-network-common_2.12-3.0.0.jar`

Reply:
Sorry for forgetting update the README. Currently I used the spark3.0.0.